src/NUnitFramework/framework/Internal/Tests/TestFixture.cs

```diff
@@ -31,6 +31,11 @@ namespace NUnit.Framework.Internal
     /// </summary>
     public class TestFixture : TestSuite, IDisposableFixture
     {
+        /// <summary>
+        /// The LifeCycle specified for the current test fixture.
```

Comment:
Another example of where English words seem more consistent and less awkward: `The life cycle specified for the current test fixture.`

(There's wouldn't really be much reason to use `<see cref="LifeCycle"/>` since the property type is already being shown.)